deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/SpecItemConfig.java

```diff
@@ -1,14 +1,24 @@
 package io.quarkiverse.openapi.generator.deployment;
```

Comment:
I do not think this class is needed any longer. 
I just try deleting it and the mvn clean install still works

Reply:
It's there for doc config / quarkus bind purposes only. In your use case you won't see the warning because you don't have a property in your env prefixing with "quarkus.openapi-generator.codegen.spec", you won't see it, but end users will.